Code/Framework/AzToolsFramework/AzToolsFramework/API/EditorCameraBus.h

```diff
@@ -116,6 +135,16 @@ namespace Camera
          */
         virtual void ToggleCameraAsActiveView() = 0;
 
+        /**
+         * Aligns this camera with the active view in the scene and sets it as the active camera
```

Comment:
uber nit: Missing period (`.`) at the end (same below too) - also maybe something for a brief follow-up PR, but it would be nice to update all the comments to the `//!` style as they're much easier to maintain. Not a huge deal at all just something I noticed (if it adds too much noise to this PR don't worry about it, could be a separate follow-on one if you have time 😝)